src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -263,7 +269,7 @@ public int getPendingCompactions() {
     checkNotNull(cmProxy, "Looks like the proxy is not connected");
     try {
         ObjectName name = new ObjectName(COMP_OBJECT_NAME);
-        int pendingCount = (int) mbeanServer.getAttribute(name, "PendingTasks");
+        int pendingCount = (int) mbeanServer.getAttribute(name, "Value");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Value" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)